### PR TITLE
TODOs to do

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: src/main/java/org/fundacionjala/pivotal/pageobjects/login/SignInPage.java [hunk 1]
```diff
@@ -84,6 +84,7 @@ public static Dashboard loginAs(final String username, final String password) {
      * @param password that.
      * @return a Dashboard object
      */
+    //TODO join both methods
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/pivotal/pageobjects/login/SignInPage.java:87]: INFO Complete the task associated to this TODO comment. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)

File: src/main/java/org/fundacionjala/core/ui/browsers/GoogleChrome32.java [hunk 2]
```diff
@@ -15,6 +15,7 @@
      * Empty constructor and path fof driver.
      */
     public GoogleChrome32() {
+        //TODO multi platform support
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/core/ui/browsers/GoogleChrome32.java:18]: INFO Complete the task associated to this TODO comment. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)

File: src/test/java/org/fundacionjala/pivotal/cucumber/steps/ProjectSteps.java [hunk 3]
```diff
@@ -3,6 +3,7 @@
 import cucumber.api.java.en.And;
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/pivotal/cucumber/steps/ProjectSteps.java:3]: MINOR Remove this unused import 'cucumber.api.java.en.And'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/main/java/org/fundacionjala/core/ui/CommonActions.java [hunk 4]
```diff
@@ -21,6 +21,7 @@ private CommonActions() {
      * wait and click action.
      * @param element webelement.
      */
+    //TODO rename methods
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/core/ui/CommonActions.java:24]: INFO Complete the task associated to this TODO comment. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)

File: src/main/java/org/fundacionjala/pivotal/pageobjects/dashboard/IFormFields.java [hunk 5]
```diff
@@ -8,5 +8,6 @@
     /**
      * Method to be modify by lambda.
      */
+    //TODO rename method to execute steps
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/pivotal/pageobjects/dashboard/IFormFields.java:11]: INFO Complete the task associated to this TODO comment. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)

File: src/main/java/org/fundacionjala/core/util/Environment.java [hunk 6]
```diff
@@ -35,6 +35,7 @@ private void readFile() {
             properties.load(input);
         } catch (IOException ex) {
             LOGGER.error(ex.getMessage());
+            //TODO stop execution with message
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/core/util/Environment.java:38]: INFO Complete the task associated to this TODO comment. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)

File: src/test/java/org/fundacionjala/pivotal/cucumber/runner/RunCucumberTest.java [hunk 7]
```diff
@@ -1,7 +1,11 @@
 package org.fundacionjala.pivotal.cucumber.runner;
 
 import cucumber.api.CucumberOptions;
+import cucumber.api.java.After;
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/pivotal/cucumber/runner/RunCucumberTest.java:4]: MINOR Remove this unused import 'cucumber.api.java.After'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/test/java/org/fundacionjala/pivotal/cucumber/runner/RunCucumberTest.java [hunk 8]
```diff
@@ -1,7 +1,11 @@
 package org.fundacionjala.pivotal.cucumber.runner;
 
 import cucumber.api.CucumberOptions;
+import cucumber.api.java.After;
 import cucumber.api.testng.AbstractTestNGCucumberTests;
+import org.fundacionjala.core.ui.WebDriverManager;
+import org.openqa.selenium.WebElement;
```
Comment (carledriss) [inline src/test/java/org/fundacionjala/pivotal/cucumber/runner/RunCucumberTest.java:7]: MINOR Remove this unused import 'org.openqa.selenium.WebElement'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

File: src/main/java/org/fundacionjala/pivotal/pageobjects/login/HomePage.java [hunk 9]
```diff
@@ -15,6 +15,7 @@
     //Page URL
     @FindBy(how = How.LINK_TEXT, using = "Sign in")
     private WebElement signin;
+    //TODO
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/pivotal/pageobjects/login/HomePage.java:18]: INFO Complete the task associated to this TODO comment. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)

File: src/main/java/org/fundacionjala/pivotal/pageobjects/dashboard/FormCreate.java [hunk 10]
```diff
@@ -116,6 +116,7 @@ public void setProjectAccount(final String accountName) {
      * @param value .
      */
     public void setProjectPrivacy(final String value) {
+        //TODO change to if
```
Comment (carledriss) [inline src/main/java/org/fundacionjala/pivotal/pageobjects/dashboard/FormCreate.java:119]: INFO Complete the task associated to this TODO comment. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1135)

**Comment by carledriss** [comment]
SonarQube analysis reported 12 issues
* ![MINOR][MINOR] 5 minor
* ![INFO][INFO] 7 info

Watch the comments in this conversation to review them.

#### 2 extra issues

Note: The following issues were found on lines that were not modified in the pull request. Because these issues can't be reported as line comments, they are summarized here:

1. ![MINOR][MINOR] [AbstractPage.java#L6](https://github.com/AWT-01/pivotal-selenium-tests/blob/3503b24e5778446b6afbebb6e2c7c3be3b0b4ef8/src/main/java/org/fundacionjala/core/ui/AbstractPage.java#L6): Remove this unused import 'org.openqa.selenium.support.events.AbstractWebDriverEventListener'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)
1. ![MINOR][MINOR] [Hooks.java#L5](https://github.com/AWT-01/pivotal-selenium-tests/blob/3503b24e5778446b6afbebb6e2c7c3be3b0b4ef8/src/test/java/org/fundacionjala/pivotal/hooks/Hooks.java#L5): Remove this unused import 'org.fundacionjala.pivotal.pageobjects.dashboard.Dashboard'. [rule](https://sonarcloud.io/coding_rules#rule_key=squid%3AUselessImportCheck)

[INFO]: https://sonarsource.github.io/sonar-github/severity-info.png 'Severity: INFO'
[MINOR]: https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR'

**Review by 11TK11 — APPROVED** [review]

**Pull request merged**